descriptor.json

```diff
@@ -0,0 +1,29 @@
+{
+        "package": {
+        "name": "vaani.iot", 
+        "repo": "openhab-distro",
+        "subject": "vaani/vaani.iot",
+        "desc": "Vaani OpenHab submodule",
+        "website_url": "www.vaani.iot",
+        "vcs_url": "https://github.com/mozilla/openhab-distro.git",
+        "github_use_tag_release_notes": true,
+        "github_release_notes_file": "README.md",
+        "labels": ["vaani", "openhab", "mozilla"],
+        "public_download_numbers": false,
+        "public_stats": false,
+        },
+
+
+    "version": {
+        "name": "0.1"
+    },
+
+
+
+    "files":
+        [
+        {"includePattern": "build/bin(.*)*/(.*\.gem)", "excludePattern": ".*/do-not-deploy/.*", "uploadPattern": "gems/$2"},
```

Comment:
Following up [on the parsing error](https://github.com/mozilla/openhab-distro/pull/1/files#r56628540).

After putting this json file in [a json linter](http://jsonlint.com), this particular line is not valid. I think what Github pointed out is correct. This has to be checked with the documentation of Bintray, though.

cc @yglazko